_includes/pbc/all/install-features/202304.0/install-warehouse-user-management-feature.md

```diff
@@ -0,0 +1,319 @@
+| UserCriteria.userReference                      | property | deprecated | src/Generated/Shared/Transfer/UserCriteria                                    |
+| UserCriteria.withExpanders                      | property | deprecated | src/Generated/Shared/Transfer/UserCriteria                                    |
+| StockCriteriaFilter.idStock                     | property | deprecated | src/Generated/Shared/Transfer/StockCriteriaFilter                             |
+
+{% endinfo_block %}
+
+### 3) Set up configuration
+
+If you want to make `warehouse-user-assignments` resource protected, adjust the protected paths configuration:
+
+**src/Pyz/Shared/GlueBackendApiApplicationAuthorizationConnector/GlueBackendApiApplicationAuthorizationConnectorConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Shared\GlueBackendApiApplicationAuthorizationConnector;
+
+use Spryker\Shared\GlueBackendApiApplicationAuthorizationConnector\GlueBackendApiApplicationAuthorizationConnectorConfig as SprykerGlueBackendApiApplicationAuthorizationConnectorConfig;
+
+class GlueBackendApiApplicationAuthorizationConnectorConfig extends SprykerGlueBackendApiApplicationAuthorizationConnectorConfig
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function getProtectedPaths(): array
+    {
+        return [
+            '/\/warehouse-user-assignments(?:\/[^\/]+)?\/?$/' => [
+                'isRegularExpression' => true,
+            ],
+        ];
+    }
+}
+
+```
+
+### 4) Add translations
+
+1. Append glossary according to your configuration:
+
+```csv
+warehouse_user_assignment.validation.user_not_found,User not found.,en_US
+warehouse_user_assignment.validation.user_not_found,Benutzer nicht gefunden.,de_DE
+warehouse_user_assignment.validation.warehouse_not_found,Warehouse not found.,en_US
+warehouse_user_assignment.validation.warehouse_not_found,Lager nicht gefunden.,de_DE
+warehouse_user_assignment.validation.warehouse_user_assignment_not_found,Warehouse user assignment not found.,en_US
+warehouse_user_assignment.validation.warehouse_user_assignment_not_found,Lagerbenutzerzuweisung nicht gefunden.,de_DE
+warehouse_user_assignment.validation.too_many_active_warehouse_assignments,User has too many active warehouse assignments.,en_US
+warehouse_user_assignment.validation.too_many_active_warehouse_assignments,Dem Benutzer sind zu viele aktive Läger zugewiesen.,de_DE
+warehouse_user_assignment.validation.warehouse_user_assignment_already_exists,Warehouse user assignment already exists.,en_US
+warehouse_user_assignment.validation.warehouse_user_assignment_already_exists,Lagerbenutzerzuweisung existiert bereits.,de_DE
+```
+
+2. Import data:
+
+```bash
+console data:import glossary
+```
+
+3. Add Zed translations:
+
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+{% info_block warningBox "Verification" %}
```

Comment:
I've rewritten the Verification steps, so review them and make sure they are correct since I don't have this feature to be able to test it in the Back Office on my own.

Reply:
Looks good to me 👍